src/html/template.html

```diff
@@ -0,0 +1,123 @@
+       <div class="fl-mapviz-outer-container">
```

Comment:
As the framework upgrade no longer supports the load of markup from a template, can this template file be removed? When the time comes that the new framework supports the template loading, this file can be recreated by copying the markup from `index.html` that is currently in use.

Reply:
The framework doesn't support dynamic loading at the client, but I believe this file could still be used statically on the server by means of a nunjucks include or so?

Reply:
Thanks for the clarification. I see your point which makes sense.